rules/falco_rules.yaml

```diff
@@ -1544,9 +1544,9 @@
     an attempt to change a program/thread\'s namespace (commonly done
     as a part of creating a container) by calling setns.
   condition: >
-    evt.type = setns
-    and not proc.name in (docker_binaries, k8s_binaries, lxd_binaries, sysdigcloud_binaries,
-                          sysdig, nsenter, calico, oci-umount, cilium-cni, network_plugin_binaries)
+    evt.type=setns and evt.dir=<
+    and not (container.id=host and proc.name in (docker_binaries, k8s_binaries, lxd_binaries, nsenter))
+    and not proc.name in (sysdigcloud_binaries, sysdig, calico, oci-umount, cilium-cni, network_plugin_binaries)
```

Comment:
This line was conflicted because of the PR (https://github.com/falcosecurity/falco/commit/48a0f512fb9e5022135ca42040242bef19a98aea), and I fixed the conflict.

Reply:
did you rebase?

Reply:
Yes, after I got needs-rebase label from poiana. The point I changed in first commit is to add cilium-cni.